src/synapsepm/utils/Utils.php

```diff
@@ -8,7 +8,7 @@
 
 class Utils {
 
-    const NUKKIT_RUNTIMEID_TABLE = "https://raw.githubusercontent.com/NukkitX/Nukkit/master/src/main/resources/runtimeid_table.json";
+    const NUKKIT_RUNTIMEID_TABLE = "http://host.firebt.net/blocks.json";
```

Comment:
Umm... Shouldn't we be using the original?

Reply:
With the original it generates a bug in legacy_id for now you can use my website as you repair that failure of yours